### DISCUSSION TITLE
Modified time update

### DISCUSSION THREAD

**Discussion by darrepac** [abraunegg/onedrive#1457] [Q&A] [answered]
Hello

I have modified localy the "modified time" of several files (pictures).
Seems that the client detect the difference (see below), but there is no change done/send on the remote... Is this normal?

Note: I am still uploading data, so may-be such changes will be done later on??

```
The local item has a different modified time 2020-Jul-29 16:13:51Z when compared to remote modified time 2021-May-04 19:55:40Z
The local item has a different modified time 2020-Jul-29 16:13:51Z when compared to remote modified time 2021-May-04 19:55:40Z
The local item has a different modified time 2020-Jul-29 16:13:51Z when compared to remote modified time 2021-May-04 19:55:40Z
The local item has a different modified time 2020-Jul-31 14:00:41Z when compared to remote modified time 2021-May-04 19:55:40Z
The local item has a different modified time 2020-Aug-04 15:54:10Z when compared to remote modified time 2021-May-04 19:55:41Z
```

**Comment by abraunegg** [comment, marked as answer]
@darrepac 
What should be happening is that when a file is modified locally / time is manually changed, the updated timestamp should trigger the client to use the API to update the timestamp on the remote file.

This possible *could* be a bug, but also might *not* be a bug because the OneDrive (online / remote) based on timestamp thinks it is 'newer', thus, will not update the timestamp of the remote to prevent sync issues / data-loss etc in-case the client *is* wrong about what the true state of the timestamp is.

**Reply by darrepac** [reply]
Once my upload is finished (could take some time, so may-be I will not wait for it), I will check with the windows client to see what is happening

**Reply by abraunegg** [reply]
@darrepac 
Just remember this: Microsoft wrote the API / OneDrive so that OneDrive (online) *is* the source of truth, and that source of truth is governed by hashes of the file. Microsoft did not 'envisage' their platform for users to treat their local file system as the source of truth.

**Reply by darrepac** [reply]
Interesting
But by checking with Windows client, it was to help to know if it was a bug of the linux client or just a "feature" of Onedrive

**Reply by darrepac** [reply]
Seems something crazy but I will just tell you what I see.
So the linux client is saying the messages above.
On Onedrive online, on IOS client and also on a Windows PC, the date is not changed, neither in the property, nowhere....
Yet, if I open the file on the windows PC (and it happens the same if I open it with IOS client), the date miracly change to the date I have changed (which is coming from EXIF info)... and now the date in onedrive online is correct
So to make it short, it seems that the linux client updated the creation date but to see it, you have to open the file! How strange!

**Reply by abraunegg** [reply]
@darrepac 
Thanks for the feedback - but as you can now see - the client *is* working correctly and accurately.

Please file a bug with the OneDrive API team.
